src/test/helpers/remoteRepositories.ts

```diff
@@ -0,0 +1,299 @@
+        repoData.forEach((localRepoData) => {
+            repos[localRepoData.name] = {
+                url: `file://${localRepoData.url}/.git`,
+                branch: branchName
+            };
+        });
+        EvaluateWithRemoteRepos.writeParentRepos(rootDir, repos);
+        this.updateRepo(rootDir);
+        return repos;
+    }
+
+    private branchOff(localRepoData: ILocalRepoData, releaseBranch: string): void {
+        this.commitSomeChanges(localRepoData);
+        this.checkoutBranch(localRepoData.url, releaseBranch);
+    }
+
+    private commitSomeChanges(localRepoData: ILocalRepoData, fileName: string = 'test.txt'): void {
+        fs.writeFileSync(`${localRepoData.url}/${fileName}`, 'test', 'utf8');
+        this.updateRepo(localRepoData.url);
+    }
+
+    // tslint:disable-next-line:no-any
+    private debugLog(message?: any, ...args: any[]): void {
+        if (this.debug) {
+            console.log(message, ...args);
+        }
+    }
+
+    private execSync(pathToRepo: string, command: string): void {
+        try {
+            this.debugLog(child_process.execSync(
+                command,
+                {
+                    cwd: pathToRepo,
+                    shell: 'bash'
+                }
+            ).toString());
+        } catch (e) {
+            console.log(`error when executing command [${command}] in [${pathToRepo}]!`, e);
+            throw e;
+        }
+    }
+
+    private createRepositories(repos: string[], rootDir: string): ILocalRepoData[] {
+        const command = `git init && git commit --no-gpg-sign --allow-empty -m "empty" `;
+        const data: ILocalRepoData[] = [];
+        for (const repo of repos) {
+            const pathToRepo = path.join(rootDir, repo);
+            fs.mkdirSync(pathToRepo);
+            this.debugLog(`init repo ${repo} in ${pathToRepo}`);
+            this.execSync(pathToRepo, command);
+            data.push({
+                          name: repo,
+                          url: pathToRepo
+                      });
+        }
+        return data;
+    }
+
+    private checkoutBranch(pathToRepo: string, branch: string): void {
+        const command = `git checkout -B "${branch}"`;
+        this.debugLog(`checkout branch in ${pathToRepo}: ${command}`);
+        this.execSync(pathToRepo, command);
+    }
+
+    private updateRepo(pathToRepo: string): void {
+        const command = `git status && git add '.' && git commit --no-gpg-sign --allow-empty -m "empty"`;
+        this.debugLog(`update repo in ${pathToRepo}`);
+        this.execSync(pathToRepo, command);
+    }
+
+    private tagRepo(pathToRepo: string, version: string): void {
+        const command = `git tag -a "${version}" -m "empty"`;
+        this.debugLog(`tag repo in ${pathToRepo} to ${version}`);
+        this.execSync(pathToRepo, command);
+    }
+
+    private cloneRepo(pathToRepo: string, repoUrl: string): void {
+        const command = `git clone ${repoUrl}`;
```

Comment:
```suggestion
        const command = `git clone "${repoUrl}"`;
```
By using quotes on the repo url the tests worked on windows, but only from git bash, not from windows cmd and not from within vscode.

Reply:
Works 👍 on OSX.